src/website/app/pages/Color/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { createStyledComponent } from '../../../../utils';
+import Markdown from '../../Markdown';
+import Callout from '../../Callout';
+import ColorRamp from './ColorRamp';
+import RampLegend from './RampLegend';
+// $FlowFixMe
```

Comment:
You can add something like the following to the `.flowconfig` to fix this error.

```
module.name_mapper='.*.md$' -> '<PROJECT_ROOT>/utils/emptyObject'
```

You'll then need to remove all of the `$FlowFixMe` suppressions that this fixes.